backend/app/common/CrudRepository.ts

```diff
@@ -244,8 +244,8 @@ export function CrudRepository<T extends { id: string }, Props = WithoutId<T>>(
         .returning('id')
     }
 
-    async update(id: string, data: any): Promise<void> {
-      await this.db.knex
+    async update(id: string, data: any): Promise<any> {
+      return await this.db.knex
```

Comment:
Can we keep this as void? The any type is a bit of an escape hatch, best not to use it in public interfaces.